packages/playground/README.md

```
@@ -0,0 +1,191 @@
+pnpm dev
+```
+
+The app will start at **http://localhost:5173**
+
+The mock API server runs at **http://localhost:4000**
+
+## 🎯 Features
+
+### 📁 Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)